src/ralph/settings/base.py

```diff
@@ -161,6 +160,8 @@ def _gen():
 
 WSGI_APPLICATION = "ralph.wsgi.application"
 
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
Generally big auto field is preferred but it would create many migrations - we can do that later